packages/venia-concept/src/services/restService/restService.js

```diff
@@ -0,0 +1,29 @@
+import { RestApi } from '@magento/peregrine';
+import purchaseHistoryMock from './mocks/purchaseHistoryMock';
+
+const { request } = RestApi.Magento2;
+
+const API_BASE = `/rest/V1`;
+
+export const addItemToGuestCart = cartItem =>
+    request(`${API_BASE}/guest-carts/${cartItem.quote_id}/items`, {
+        method: 'POST',
+        body: JSON.stringify({
+            cartItem
+        })
+    });
+
+// TODO: implement method.
+export const fetchPurchaseHistory = () =>
+    new Promise(resolve => {
+        setTimeout(() => resolve(purchaseHistoryMock), 1000);
+    });
+
+const restService = {};
```

Comment:
I would just leave out the `api` object and put the action creators directly on the default export object.

```suggestion
const restService = {
  addItemToGuestCart,
  fetchPurchaseHistory
};
```